.travis.yml

```diff
@@ -110,6 +110,15 @@ jobs:
     - BENCHMARK_REPEATS=1 make benchmark-all
     after_success: skip
 
+  - stage: test
+    python: 3.7
+    name: 'Pydantic-Only Benchmarks'
+    script:
+    - pip install -r benchmarks/requirements.txt
+    - make build-cython
```

Comment:
I think this can be skipped for performance.

Reply:
This was copied directly from the Benchmarks test above. If it's not needed for pydantic-only, can certainly be removed.